packages/integration-tests/src/agent-zai.integration.test.ts

```
@@ -0,0 +1,167 @@
+    expect(aliveRunning).toBe(true);
+  });
+
+  it("getActivityState → returns valid state while running", () => {
+    expect(aliveActivityState).toBeDefined();
+    if (aliveActivityState) {
+      expect(aliveActivityState.state).not.toBe("exited");
+    }
+  });
+
+  it("getSessionInfo → returns session data while running (or null)", () => {
+    if (aliveSessionInfo != null) {
```

ESLint

[error] Expected '!==' and instead saw '!='.

(eqeqeq)